### DISCUSSION TITLE
Results from security collections

### DISCUSSION THREAD

**Discussion by AGlennDa** [trimble-oss/dba-dash#1017] [Q&A] [answered]
Hi, 
I am trying to create some documentation on how to see the results of all the good collections in DBA Dash. 
I am now looking at the ServerPrincipals, ServerRoleMembers, ServerPermissions, DatabasePrincipals, DatabaseRoleMembers and DatabasePermissions. The collections are running just fine - The tables corresponding to the collections inside of the repository db is being populated. But where do I see the results of this in the DBA Dash GUI?

**Comment by DavidWiseman** [comment, marked as answer]
Hi,

These collections are not displayed anywhere in the GUI.  They were originally included in some SSRS reports but the reports never made their way into the GUI.

I have a custom report to display sysadmin users which I've shared here if you want to display this in the GUI.

https://github.com/trimble-oss/dba-dash/discussions/1018

If you are not interested in these collections it would also be safe to disable them.  

Hope this helps,

David

**Reply by AGlennDa** [reply]
Hi David, 
Thank you for the fast response :) It is much appreciated. 
I will look into the custom report. Thank you for that.
